rmw_fastrtps_shared_cpp/CMakeLists.txt

```diff
@@ -49,6 +49,9 @@ find_package(fastcdr REQUIRED CONFIG)
 find_package(fastrtps 2.3 REQUIRED CONFIG)
 find_package(FastRTPS 2.3 REQUIRED MODULE)
 
+find_package(Threads REQUIRED)
+find_package(tracy_vendor REQUIRED)
```

Comment:
What's with the `tracy_vendor` here?  Do we need this to get this work in?

Reply:
No, there are some profiling points in this branch, I will remove for final merge if we determine this is a valid approach.